pxr/imaging/hgi/graphicsPipeline.h

```diff
@@ -366,14 +366,18 @@ struct HgiTessellationState
 {
     enum PatchType {
         Triangle,
-        Quad
+        Quad,
+        Isoline
     };
 
     HGI_API
     HgiTessellationState();
 
     PatchType patchType;
     int primitiveIndexSize;
+    int numVertsOut;
+    bool useConstantTessFactors;
+    bool isPostTessControl = false;
```

Comment:
Make sure these are being set upstream and used downstream (e.g. see notes in hgiMetal/graphicsPipeline.mm)
Possibly enumerating tess factor modes...?